src/schema.cpp

```diff
@@ -199,7 +199,8 @@ const char* MODELS_CONFIG_SCHEMA = R"({
 				},
 				"demultiply_count": {
 					"type": "integer",
-					"minimum": 2
+					"minimum": 2,
+					"maximum": 100000
```

Comment:
100k is quite large value. For 2 levels of demultiplexing of max value 100k we have 100k*100k inferences = 10 billion inferences. It would be more reasonable to set max to something like 1000? @atobiszei

Reply:
I can imagine a photo of grid with 100x100. Initially I thought this is enough to cover most of cases but now I am actually worried that this may be to low. Maybe we should not limit this but just print warning if we will see from config that this pipeline will schedule >10k inferences? 

This way we do not limit functionality but warn admin about possible issue.

Reply:
Let's stay with 10k for now - we can relax this limitation further on.